ui/apps/platform/cypress/integration/networkGraphPF/networkGraph.helpers.js

```diff
@@ -63,18 +63,15 @@ export function selectDeployment(deployment) {
 
 export const notifiersAlias = 'notifiers';
 export const clustersAlias = 'clusters';
-export const networkPoliciesGraphEpochAlias = 'networkpolicies/graph/epoch';
+// Removed the following because request has 30 second delay from polling interval:
```

Comment:
👍 

FWIW we also receive the `epoch` value from the main request that retrieves the network graph info once a cluster is selected. This value is currently unimplemented (always returns `0`), but if/when it is implemented we might be able to use it as a reliable way to tell that the graph has "settled".